WootricSDK/WootricSDK/WTRApiClient.m

```diff
@@ -274,7 +274,7 @@ - (void)checkEligibility:(void (^)())eligible {
 
   NSURL *url = [NSURL URLWithString:baseURLString];
   NSMutableURLRequest *urlRequest = [NSMutableURLRequest requestWithURL:url];
-  [urlRequest setValue:@"Wootric-Mobile-SDK" forHTTPHeaderField:@"USER_AGENT"];
+  [urlRequest setValue:@"Wootric-Mobile-SDK" forHTTPHeaderField:@"User-Agent"];
```

Comment:
Uh... I wish we had a test to prevent this from happening again. Will it be possible to add it?